Sources/AblyAssetTrackingPublisher/WorkerQueue/PublisherWorkerFactory.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import AblyAssetTrackingInternal
+
+internal class PublisherWorkerFactory: WorkerFactory
+{
+    public typealias PropertiesType = PublisherWorkerQueueProperties
```

Comment:
I don't think these `typealias` are necessary – the compiler should infer them from the signature of `createWorker`.

Reply:
86abd82